libass/ass_render.c

```diff
@@ -1632,7 +1632,8 @@ static void trim_whitespace(ASS_Renderer *render_priv)
  * FIXME: implement style 0 and 3 correctly
  */
 static void
-wrap_lines_smart(ASS_Renderer *render_priv, double max_text_width)
+wrap_lines_smart(ASS_Renderer *render_priv, double max_text_width,
+                 bool always_wrap)
```

Comment:
Better to synchronize this name with the feature name (`wrap_on_overflow` or something).

Reply:
The `smart` comes from ASS, where wrap modes `0` and `3` are referred to as "smart-wrapping". I think it's better to keep this name. Also when Unicode line-breaking gets merged, `on_overflow` would no longer be the only Feature to influence this function's behaviour.

Reply:
I think the earlier comment was about the new parameter’s name, rather than the function’s name.

Reply:
> I think the earlier comment was about the new parameter’s name

Ahh, that makes sense `^^. Then just ignore my previous comment.